gdx-jnigen-gradle/src/main/resources/META-INF/gradle-plugins/com.badlogicgames.gdx.gdx-jnigen.properties

```diff
@@ -0,0 +1 @@
+implementation-class=com.badlogic.gdx.jnigen.gradle.JnigenPlugin
```

Comment:
I think the file name is supposed to be "jnigen.properties", since that's the id of the plugin

Reply:
Only official gradle plugins are allowed single word plugin names. The rest are classpath-like.

so we would have `apply plugin: "com.badlogicgames.gdx.gdx-jnigen-gradle"`